src/components/CategoryFilter.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { catFilter } from '../actions/index';
+
+const mapDispatchToProps = dispatch => ({
+  catFilter: filter => {
+    dispatch(catFilter(filter));
+  },
+});
+
+const CategoryFilter = ({ catFilter }) => {
```

Comment:
Kindly change this component to a presentational component

Reply:
Done